support/client/lib/vwf/model/jPlayer.js

```diff
@@ -342,6 +360,13 @@ define( [
 
     } );
 
+    function videoEndedCallback(){
+        var mediaManagerID = vwf.find( undefined, "/mediaManager" )[ 0 ];
+        var videoManagerID = vwf.find( mediaManagerID, "videoManager" ) [ 0 ];
+        console.log("Video ended callback in driver fired!");
+        vwf.fireEvent(videoManagerID, "videoEnded");
```

Comment:
this.kernel instead of vwf

On Fri, Feb 13, 2015 at 11:51 AM, Nathan Marshak <notifications@github.com>
wrote:

> In support/client/lib/vwf/model/jPlayer.js
> <https://github.com/virtual-world-framework/vwf/pull/455#discussion_r24677781>
> :
>
> > @@ -342,6 +360,13 @@ define( [
> >
> >      } );
> >
> > +    function videoEndedCallback(){
> > +        var mediaManagerID = vwf.find( undefined, "/mediaManager" )[ 0 ];
> > +        var videoManagerID = vwf.find( mediaManagerID, "videoManager" ) [ 0 ];
> > +        console.log("Video ended callback in driver fired!");
> > +        vwf.fireEvent(videoManagerID, "videoEnded");
>
> If it's not appropriate, what's the best alternative? @BrettASwift
> <https://github.com/BrettASwift> @davideaster
> <https://github.com/davideaster> @eric79 <https://github.com/eric79>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/virtual-world-framework/vwf/pull/455/files#r24677781>.
>